pystiche_papers/sanakoyeu_et_al_2018/_loss.py

```diff
@@ -289,3 +293,94 @@ def transformed_image_loss(
     return ops.FeatureReconstructionOperator(
         transformer_block, score_weight=score_weight
     )
+
+
+class FeatureReconstructionOperator(ops.FeatureReconstructionOperator):
+    def __init__(
+        self,
+        encoder: SequentialEncoder,
+        score_weight: float = 1.0,
+        impl_params: bool = True,
+    ):
+        super().__init__(encoder, score_weight=score_weight)
+        self.impl_params = impl_params
```

Comment:
Never store `impl_params`, but only store the changes. For example here it could be `reduction = "mae" if impl_params else "mse"` or something similar. Don't do this though and look at the next comment first.